CHANGELOG.md

```diff
@@ -4,16 +4,16 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
-## [1.12.0-rc1](https://github.com/go-gitea/gitea/releases/tag/v1.12.0-rc1) - 2020-05-18
+## [1.12.0-rc2](https://github.com/go-gitea/gitea/releases/tag/v1.12.0-rc2) - 2020-06-08
```

Comment:
Thanks for PR :)
For RC changlogs they are usually separated by RC so users can see what has changed in each individual RC, then when stable comes then they are combined.

Reply:
uff OK will seperate